doc/api/readline.md

```diff
@@ -527,11 +528,113 @@ rl.on('line', (line) => {
 });
 ```
 
+## TTY keybindings
+
+<table>
+  <tr>
+    <th>Keybindings</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>ctrl+shift+backspace</code></td>
```

Comment:
Suggestion: add whitespace between the individual commands and maybe also only highlight the keys, not the plus. As in: `<code>ctrl</code> + <code>shift</code> + <code>backspace</code>`

Reply:
Any reason these are wrapped in `<code>` and not [`<kbd>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd)?